### PR TITLE
Add unittests for VBitVec

### PR DESCRIPTION
**Pull request by WTer**

### CODE REVIEW

File: p2p/vbitvec.go [hunk 1]
```diff
@@ -127,17 +135,17 @@ func (v *VBitVec) has1fecBit(s *VBitVec) bool {
     return diffCnt == 1
 }
 
-//getBitsCnt calculate bit num
+// getBitsCnt Calculate bit num
```
Comment (herove) [inline p2p/vbitvec.go]: Commens should be a sentence,eg: getBitsCnt calculates bit num

File: p2p/vbitvec_test.go [hunk 2]
```diff
@@ -26,5 +28,140 @@ func Test_VBitVec_check(t *testing.T) {
...
+    v4 := new(VBitVec)
+    v4.Init(64)
+    v4.SetBit(2, true)
+    v2.BitXor1(v4)
+    if !bytes.Equal(v2.pBuf, v4.pBuf) {
+        fmt.Println("Detach pBuf:", v2.pBuf, v4.pBuf)
+        t.Fail();
+    }
+
+}
```
Comment (herove) [inline p2p/vbitvec_test.go]: No newline and end of file.

File: p2p/vbitvec.go (hunk 1 above)
Comment (WTer) [reply p2p/vbitvec.go]: nice suggestion.

File: p2p/vbitvec_test.go (hunk 2 above)
Comment (WTer) [reply p2p/vbitvec_test.go]: added a newline ad the end of this file.

**Pull request merged**